crates/wasm_module/src/opcodes.rs

```diff
@@ -232,6 +233,7 @@ fn immediates_for(op: OpCode) -> Result<OpImmediates, String> {
         | I64STORE32 => Leb32x2,
 
         CURRENTMEMORY | GROWMEMORY => Byte1,
+        MEMORY => Leb32x2,
```

Comment:
this is almost certainly wrong.

 I don't fully understand what immediates are I think. if it means the total width of the instruction arguments, we have a problem here, because `memory.fill` and `memory.copy` have different numbers of arguments (`0xfc 0x0b 0x00` for memory.fill, `0xfc 0x0a 0x00 0x00` for memory.copy). so we need the second byte to know how wide the instruction is.